discovery-infra/start_discovery.py

```diff
@@ -259,6 +303,8 @@ def nodes_flow(client, cluster_name, cluster, image_path):
     if cluster:
         nodes_details["cluster_inventory_id"] = cluster.id
 
+    nodes_details.update(_secondary_network_details())
```

Comment:
why not to create second network in the _create_node_details function. Second network is not going to be optional, this is our default configuration

Reply:
correct, yet we need to support multiple environments, so we need to generate the parameters for each environment to avoid conflicts.

Reply:
I realized in delay what you meant, fixed